src/MarginTrading.MarketMaker/Startup.cs

```diff
@@ -208,6 +211,8 @@ private static ILog CreateLogWithSlack(IServiceCollection services, IReloadingMa
                     consoleLogger);
 
                 azureStorageLogger.Start();
+                
+                services.AddSingleton(azureStorageLogger);
```

Comment:
Why you need this log, but don't use aggregateLogger as ILog in Trace?

Reply:
This is correct, but a bit complicated. Maybe we should discuss it using voice.